tests/acceptance/bootstrap/SharingNgContext.php

```diff
@@ -508,6 +508,24 @@ public function userSendsTheFollowingSpaceShareInvitationToFederatedUserUsingPer
 		);
 	}
 
+	/**
+	 * @Given user :user has updated the last resource share with:
+	 *
+	 * @param string $user
+	 * @param TableNode $table
+	 *
+	 * @return void
+	 */
+	public function userHasUpdatedTheLastResourceShareWith(string $user, TableNode $table): void {
+		$permissionID = $this->featureContext->shareNgGetLastCreatedUserGroupShareID();
+		$response = $this->updateResourceShare(
+			$user,
+			$table,
+			$permissionID
+		);
+		$this->featureContext->theHTTPStatusCodeShouldBe(200, "", $response);
```

Comment:
```suggestion
		$this->featureContext->theHTTPStatusCodeShouldBe(200, "", $response);
```
why empty string is passed if its message parameter, then i don't think passing proper message could be problem

Reply:
I have seen same case in other places as well. so i did the same.
I will pass a proper message :+1:

Reply:
discuss with **person-in-charge** and do as he suggest to make refactor PR or let it slide